README.md

```diff
@@ -250,7 +261,7 @@ default, we use the diesel test_transaction functionality to ensure test data
 is not committed to the database. Therefore, there is an environment variable
 which can be used to turn off test_transaction.
 
-        SYNC_SYNCSTORAGE__DATABASE_USE_TEST_TRANSACTIONS=false cargo test [testname]
+        SYNC_SYNCSTORAGE__DATABASE_USE_TEST_TRANSACTIONS=false make test ARGS="[testname]"
```

Comment:
This is one of those weird cases where we're using a pseudo-variable `[testname]`, and I wonder if that might screw up folk. 

`cargo test` (and presumably `cargo nextest`) allow you to run tests that match a name pattern you provide. So something like `cargo nextest test_uaid` would run any test that matches `.*test_uaid.*`  Passing the value as `cargo nextest [test_uaid]` would generate an error.

We probably don't need to address it here, but we should have some consistent way to express variables in documentation like this. Maybe use `example_test_name` or something.